Makefile

```diff
@@ -59,11 +56,11 @@ verify-scripts:
 	bash -x hack/verify-crds.sh
 	bash -x hack/verify-codegen.sh
 .PHONY: verify-scripts
-verify: verify-scripts verify-codegen-crds
+verify: verify-scripts
 
 update-scripts:
 	hack/update-deepcopy.sh
 	hack/update-swagger-docs.sh
 	hack/update-codegen.sh
 .PHONY: update-scripts
-update: update-scripts update-codegen-crds
+update: update-scripts
```

Comment:
I do not think we should delete the update-codegen-crds, because the update target is designed to update all.

Reply:
When I do some changes for the API, I only want to execute one command, `make update`(similar with openshift/release, ocm/api), if we remove the update-codegen-crds for update, I need to execute `make update` and `make update-codegen-crds`, right?